targets/XPlatCppSdk/source/cppsdk/XPlatCppWindows.vcxproj.filters.ejs

```diff
@@ -17,45 +17,22 @@
     </Filter>
   </ItemGroup>
   <ItemGroup>
-    <ClInclude Include="include\playfab\PlayFabEntityDataModels.h">
-      <Filter>Header Files\playfab</Filter>
-    </ClInclude>
-    <ClInclude Include="include\playfab\PlayFabHttp.h">
```

Comment:
You should not have removed this one.
Are you sure this compiled and ran?
You can't run the unit tests, but you CAN run the Windows Example Project.

Reply:
It's still there (check out line 30).

Reply:
resolved. Thanks. Just missed it.